webpack.config.js

```diff
@@ -55,3 +72,27 @@ appAdminConfig.externals = Object.assign({}, appAdminConfig.externals, { window:
 appAdminConfig.name = 'app.admin';
 
 module.exports = [shopConfig, adminConfig, appShopConfig, appAdminConfig];
+
+/**
+ * Merge controllers.json from multiple files into one and store in cache
+ * Used to merge both controllers.json fed by packages and the one from the app
+ */
+function mergeControllers(name, filePaths, cacheDir = 'var/cache/webpack') {
```

Comment:
Do we need this functionality? Can't we just specify same deps for admin and shop?